server/db/schema.sql

```diff
@@ -0,0 +1,33 @@
+
+DROP TABLE IF EXISTS pet_types;
+CREATE TABLE pet_types (
+id SERIAL PRIMARY KEY,
+name VARCHAR(255) NOT NULL,
+img_url VARCHAR(255) NOT NULL,
+description TEXT
+);
+

```

Comment:
This schema looks great guys! 🎉 

From a style standpoint, I'd just make sure that we're consistent with our indentation. I do like the use of indentation within the `CREATE TABLE` on line 11, so let's use that as the model. Finally, make sure everyone in the group has their indentation equal to two spaces. In your `.prettierrc` you can add `"tabWidth": 2` as another key/value pair.